quantum/process_keycode/process_unicode_common.c

```diff
@@ -153,3 +143,50 @@ void send_unicode_hex_string(const char *str) {
     str += n; // Move to the first ' ' (or '\0') after the current token
   }
 }
+
+bool process_record_unicode_common(uint16_t keycode, keyrecord_t *record) {
+  if (record->event.pressed) {
+    switch (keycode) {
+      case UNICODE_MODE_OSX:
+        set_unicode_input_mode(UC_OSX);
+        #if defined(AUDIO_ENABLE) && defined(UNICODE_OSX_SONG)
+          PLAY_SONG(UNICODE_OSX_SONG);
+        #endif
+        break;
+      case UNICODE_MODE_LNX:
+        set_unicode_input_mode(UC_LNX);
+        #if defined(AUDIO_ENABLE) && defined(UNICODE_LINUX_SONG)
+          PLAY_SONG(UNICODE_LINUX_SONG);
```

Comment:
Shouldn `PLAY_SONG` use the arrays declared above instead of the constants?
```c
PLAY_SONG(song_lnx);
```

Reply:
Shoot, yeah, 

```suggestion
          PLAY_SONG(song_lnx);
```